content/getting-started/yaml.md

```diff
@@ -19,6 +19,10 @@ Builds can be also started manually by clicking **Start new build** in Codemagic
 
 ## Syntax
 
+{{<notebox>}}
```

Comment:
I would put this box after the next sentence. First we say it's yaml, then we link to how you validate it.